tests/DoctrineTestCase.php

```diff
@@ -33,6 +33,37 @@ class DoctrineTestCase extends TestCase
      */
     private $seeded = false;
 
+    /**
+     * Lazy load database schema only when required
+     *
+     * @return void
+     */
+    protected function createSchema(): void
```

Comment:
Why is this method duplicated in here (it looks like the same method is in the TestCase you've added as well?)

Reply:
These are two different test cases - one which is used for testing entities and the entity traits (see: HasProviderTest and HasProviderTestCase), which uses the `Doctrine\ORM` namespace, whereas the other test case utilises the `Eoneopay\Externals\ORM` namespace.